open-craft/opencraft
# [BB-2444] Remove public contact email from registration

**Issue**: [BB-2444](https://tasks.opencraft.com/browse/BB-2444)

**Screenshots**(Updated):

![image](https://user-images.githubusercontent.com/8607745/82651220-7ea47f00-9c39-11ea-92f5-3138a56ae722.png)

![image](https://user-images.githubusercontent.com/8607745/82651244-882de700-9c39-11ea-83c1-1619536b5933.png)


Testing instructions:
* Go through the registration process by creating a new instance, check it works perfectly or not
* At the congratulation page, the message should be right.
* Go through by signing in and checking that the first time the public contact email should be similar to the user email you set when creating the instance.

**Reviewers**:
